data/denominations/amenity/place_of_worship.json

```diff
@@ -0,0 +1,52 @@
+{
+  "properties": {
+    "path": "denominations/amenity/place_of_worship",
+    "skipCollection": true,
+    "exclude": {"generic": ["^place of worship$"]}
+  },
+  "items": [
+    {
+      "displayName": "Kingdom Hall of Jehovah's Witnesses",
```

Comment:
Many denominations translate their names into as many local languages as possible. The appropriate name usually varies by language more than by region. As it stands, id-tagging-schema is better suited to localization than NSI. This is already an issue for the flags tree, but the issue would be more acute for any denomination tree.

Or would we limit this tree to just the denominations that have a specialized name for all their places of worship?

Reply:
Yeah I was also thinking about how the displayNames for these denominations could be translated to a region's local language. I agree that id-tagging-schema would be a better place to have these presets because of these current limitations - I hadn't really thought about how the default presets are localized

> Or would we limit this tree to just the denominations that have a specialized name and icon for all their places of worship?

This could work, though I can't think of any off the top of my head. I was also looking into how the [`denomination=mormon`](https://wiki.openstreetmap.org/wiki/Tag:denomination%3Dmormon) wiki page suggests to use operator tags to distinguish The Church of Jesus Christ of Latter-day Saints from other splinter groups. This particular case could work in NSI under the existing operator tree as localized operator values would be possible.

> These three presets only exist because it was considered inappropriate to use the ✝️ icon for these denominations.

This makes sense now. I was wondering why there were only these three denomination presets

Reply:
I really don't want to introduce localization into the Name Suggestion Index display names.  Display name is just the label that the preset shows in RapiD and JOSM.

But I think keeping this tree similar to how the `flags` tree works would be ok (`denomination` is an identifying value, and `name` can be whatever people want).

Reply:
> But I think keeping this tree similar to how the `flags` tree works would be ok (`denomination` is an identifying value, and `name` can be whatever people want).

#5982 tracks making flags localizable. Maybe the solution there could inform how we’d make denomination entries localizable too. But in the meantime, id-tagging-schema at least offers a way to localize the display names, even if it doesn’t have a straightforward way to filter denominations by religion (something we’d have to solve as part of this project too).